### ISSUE TITLE
Indicator click listener needed

### ISSUE THREAD

**Issue by sudarshanrao7** [Canner/android-stepsview#3]
Thanks for the library :) 
Are you planning to support indicator click listener?

**Comment by chilijung** [comment]
Yes! click listener sounds useful in many cases. I'll add it when I have time, feel free if you want to make a PR on this issue!
